### PR TITLE
[k8s] Fixing docker version in the k8s playbook

### PR DESCRIPTION
**Pull request by marianoleonardo**
* **Please check if the PR fulfills these requirements**
  * [ ] Tests for the changes have been added (for bug fixes / features)
  * [ ] Docs have been added / updated (for bug fixes / features)

* **What is the current behavior?** (You can also link to an open issue here)
When we set up the k8s cluster with the dojot playbook, the docker version that will be installed on the pods will be "latest"

* **What is the new behavior (if this is a feature change)?**
When we set up the k8s cluster with the dojot playbook, the docker version that will be installed on the pods will be `19.03.4` as specified in the k8s documentation

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

* **Is there any issue related to this PR in other repository?** (such as dojot/dojot)
Yes, this PR is connected to dojot/dojot#1898

### CODE REVIEW

**Review by andersonluisribeiro — APPROVED** [review]
:+1:

File: roles/k8s-cluster/all-nodes/tasks/main.yaml [hunk 1]
```diff
@@ -16,32 +18,40 @@
...
   apt:
     name: "{{ packages }}"
     state: present
     update_cache: yes
   vars:
     packages:
-    - docker-ce
-    - docker-ce-cli
-    - containerd.io
+    - docker-ce=5:19.03.4~3-0~ubuntu-{{ ansible_facts.lsb.codename }}
```
Comment (mprevide) [inline roles/k8s-cluster/all-nodes/tasks/main.yaml:46]: Maybe it could stay for 19.03, so minor bugs would be fixed.

Comment (LeonardoMariano) [reply roles/k8s-cluster/all-nodes/tasks/main.yaml:46]: I agree with you, however the k8s17 documentation specifies the docker version 19.03.4
`https://v1-17.docs.kubernetes.io/docs/setup/production-environment/container-runtimes/#docker`

would it be better to change to version 19.03?

Comment (mprevide) [reply roles/k8s-cluster/all-nodes/tasks/main.yaml:46]: I don't think so, if he recommends it.
@andersonluisribeiro

Comment (andersonluisribeiro) [reply roles/k8s-cluster/all-nodes/tasks/main.yaml:46]: Kubernetes release notes says it was validated in version 19.03:

https://v1-17.docs.kubernetes.io/docs/setup/release/notes/

This is the PR for kubernetes repository:

https://github.com/kubernetes/kubernetes/pull/84476

The docker 19.03.4 was released in 2019-10-17. The PR was created on 2019-10-28. I think this is the reason why they put this version in the documentation. The version 19.03.5 of the Docker was released in 2019-11-14.

Comment (andersonluisribeiro) [reply roles/k8s-cluster/all-nodes/tasks/main.yaml:46]: However I think we can use version 19.03 of the Docker.

Comment (LeonardoMariano) [reply roles/k8s-cluster/all-nodes/tasks/main.yaml]: I don't think we have this option of installing the docker in version 19.03

Following the docker documentation, either you install on the latest version or a specific version.

So, the solution adopted was to fix in version 19.03.14 which is the last bugfix of 19.03

**Review by andersonluisribeiro — APPROVED** [review]
:+1:

**Review by thiagodpf — APPROVED** [review]

**Pull request merged**